addons/docs/src/lib/docgen/createPropDef.ts

```diff
@@ -4,31 +4,45 @@ import { TypeSystem, DocgenInfo, DocgenType } from './types';
 import { JsDocParsingResult } from '../jsdocParser';
 import { createDefaultValue } from './createDefaultValue';
 
-export type PropDefFactory = (
-  propName: string,
-  docgenInfo: DocgenInfo,
-  jsDocParsingResult?: JsDocParsingResult
-) => PropDef;
+export interface PropDefFactoryProps {
+  name: string;
+  defaultPropValue: any;
+  docgenInfo: DocgenInfo;
+  jsDocParsingResult?: JsDocParsingResult;
+}
+
+export type PropDefFactory = (props: PropDefFactoryProps) => PropDef;
+
+export interface CreatePropDefProp extends PropDefFactoryProps {
+  type: DocgenType;
+}
 
-function createBasicPropDef(name: string, type: DocgenType, docgenInfo: DocgenInfo): PropDef {
+function createBasicPropDef({
+  name,
+  defaultPropValue,
+  type,
+  docgenInfo,
+}: Omit<CreatePropDefProp, 'jsDocParsingResult'>): PropDef {
   const { description, required, defaultValue } = docgenInfo;
+  const defaults = defaultValue || { value: defaultPropValue };
```

Comment:
Could you please inline that code in `createDefaultValue`. `defaults` make me thing there is multiple default values